.github/workflows/ci.yml

```
@@ -81,3 +81,17 @@ jobs:
+          set -euo pipefail
+          npm run lint
+          npm run format:check
+
+      - name: Semgrep SAST
+        run: |
+          set -euo pipefail
+          pip install semgrep --quiet
+          semgrep scan --config=auto --error \
+            clients/go clients/rust/crates conformance/runner \
+            --exclude='*.lock' --exclude='target/' 2>&1 | tail -20
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)